torch/_refs/__init__.py

```diff
@@ -667,11 +667,21 @@ def signbit(a):
     return prims.signbit(a)
 
 
+@_make_elementwise_unary_reference(ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT)
+def sgn(a):
+    return prims.sgn(a)
+
+
 @_make_elementwise_unary_reference(ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT)
 def sin(a):
     return prims.sin(a)
 
 
+@_make_elementwise_unary_reference(ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT)
+def sinc(a):
+    return prims.sinc(a)
+
+
```

Comment:
@mruberry does it make sense to implement the prim AND a ref for all these operations? I'd expect the prim to be enough.

Reply:
What about just a reference?

Reply:
This may be a good start: https://jax.readthedocs.io/en/latest/_modules/jax/_src/numpy/ufuncs.html#sinc

Reply:
Also, `sgn` may be implemented as `x / x.abs()`. But `sgn` has fast vectorised implementations, so we might want to leave it as a prim? WDYT @mruberry?

Reply:
I always thought that there were intrinsics for it because we have `Vectorized<T>::sgn()`. Now, it turns out that this is implemented in terms of `abs`, which also, as I have just learnt does not have hardware intrinsics, but it's in the STL. So yeah, completely in favour of making these two operations just references.